cluster/olm/ceph/generate-rook-csv.sh

```diff
@@ -18,13 +18,13 @@ yq="${YQ_TOOL:-yq}"
 ##########
 # CHECKS #
 ##########
-if [ ! command -v operator-sdk &>/dev/null ] && [ ! -f $operator_sdk ]; then
+if ! command -v operator-sdk &>/dev/null && [ ! -f $operator_sdk ]; then
```

Comment:
This is addressed in https://github.com/rook/rook/pull/5518 so please remove it.